packages/manager/src/features/CloudPulse/Alerts/ContextualView/AlertInformationActionRow.tsx

```diff
@@ -41,7 +52,22 @@ export const AlertInformationActionRow = (
       <TableCell sx={{ width: 0 }}>
         <FormControlLabel
           control={
-            <Toggle checked={status} onChange={() => handleToggle(alert)} />
+            <Toggle
+              checked={status}
+              disabled={isAlertActionRestricted}
+              onChange={() => handleToggle(alert)}
+              sx={(theme) => ({
+                '& .Mui-disabled+.MuiSwitch-track': {
+                  backgroundColor: theme.tokens.color.Brand[80],
+                  opacity: '0.3 !important',
+                },
+              })}
+              tooltipText={
+                isAlertActionRestricted
```

Comment:
```suggestion
                alert.group === 'account' || alert.group === 'region'
```

we are receiving alert object in this component, so no need to have additional prop passed from parent